### PR TITLE
locales(de): Add German translation

### LINKED ISSUE #8: German translation
It would be good to add the German translation (`de` locale) for the manifest.

### PR DESCRIPTION
**Pull request by jessica-jordan**
This adds the German translation for the index page content. I translated everything to my best belief and understanding as a native German speaker, but please feel free to comment if there are nuances in language or writing style that could be improved.

Resolves #8

### CODE REVIEW

File: content/_index.de.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Wir verstehen, dass die Arbeit in unserem Feld kein Recht, sondern ein Privileg ist. Der negative Einfluss toxischer Menschen am Arbeitsplatz und in der darüber hinausgehenden Gemeinschaft wird nicht durch deren technischen Beiträge wieder wettgemacht.
+
+* Wir widmen uns der Ausübung von Mitgefühl, anstelle von Verachtung. Wir weigern uns jedoch, andere Menschen aufgrund der Wahl ihrer Werkzeuge, Methoden oder Programmiersprachen herunterzumachen.
+
+* Der IT-Bereich begrüßt technischen Wandel und wird bereichert, indem es auch sozialen Wandel akzeptiert.
+
+* Wir streben danach, unsere Werte in allem, was wir tun, widerspiegeln zu lassen. Wir sind uns darüber im Klaren, dass Werte, die angepriesen, aber nicht in die Tat umgesetzt werden, in erster Linie gar keine Werte sind.
+
+
+### Unterzeichner*innen
```
Comment (jessica-jordan) [inline content/_index.de.md]: In general a lot of personal nouns will have a male (often ending in `-` or `er`, e.g. `Unterzeichner`) and a dedicated female version (often ending in `-` or `erin`, e.g. `Unterzeichnerin`) in German which is often used to describe a group of people of a particular gender. To describe groups of which the gender is not known oftentimes the _Binnen-I_ (e.g. `UnterzeichnerInnen`) is used to address people who either identify as male and female. Although this practice is very common, it's not inclusive to non-binary people imho. 

Therefore I decided to use the asterisk `*` here which can be used to infer, that not only people of male or female gender, but also people who identify with other genders are addressed. You can also read more about it in this [Wiki Article about gender-neutral language in German](https://nonbinary.miraheze.org/wiki/Gender_neutral_language_in_German):

>Nouns, Adjectives etc. [...]
Asterisk, Underscore, Punctuation
Pronounced as a pause or glottal stop. In between "feminine" and "masculine" ending, the most common version:

> mein_e beste*r Freund:in
Either after the stem - or more realistically at the next best place that has nothing to do with feminine/masculine forms but makes sense for pronunciation:

> mei:ne bes_ter Freun*din
At random places:

> mei*ne best:er Freundi_n
Can be used consistently or mixed up like in the example above.

Comment (CoralineAda) [reply content/_index.de.md]: That seems like a great approach, thank you for your sensitivity here.

File: content/_index.de.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+* Wir glauben nicht daran, dass unser Wert als Menschen intrinsisch an unseren Wert als Wissensarbeiter*innen gebunden ist. Unsere Berufe definieren uns nicht; wir sind mehr als die Arbeit, die wir ausüben.
+
+* Wir glauben daran, dass zwischenmenschliche Fähigkeiten mindestens genau so wichtig sind wie technische Fähigkeiten.
+
+* Wir können als professionelle Entwickler*innen den größten Beitrag leisten, indem wir aus der Diversität unserer Identitäten, Hintergründe, Erfahrungen und Perspektiven schöpfen. Homogenität ist ein Anti-Pattern.
+
+* Wir können erfolgreich sein und gleichzeitig vielfältige, erfüllende Leben führen. Unser Erfolg und unser Wert ist nicht davon abhängig, ob wir all unsere Energie in die Entwicklung von Software stecken.
+
+* We can be successful while leading rich, full lives. Our success and value is not dependent on exerting all of our energy on contributing to software.
```
Comment (CoralineAda) [inline content/_index.de.md]: Looks like you missed a line

File: content/_index.de.md (hunk 1 above)
Comment (jessica-jordan) [reply content/_index.de.md]: Yes, sure!

**Comment by jessica-jordan** [comment]
I just updated the PR by removing the duplicated line in the list of values

**Comment by CoralineAda** [comment]
Thank you so much!

**Pull request merged**